downstream/modules/platform/proc_configuring-tls-with-aap-operator.adoc

```diff
@@ -0,0 +1,69 @@
+////
```

Comment:
Is this a real module?

Reply:
I could have spared you the time, that file i have earmarked for deletion once all of the corrections have been made. I created it early on when we thought we needed a completely new chapter/module.